### PR TITLE
BHBC-2282: Fix draft load

### PR DESCRIPTION
**Pull request by anissa-agahchen**
(https://quartech.atlassian.net/browse/BHBC-2282)

Testing procedure:
Baseline:

Log into Dev
Throttle your network
Load a draft project
You will see an empty draft project
As a result of this code change:

Go to the app link for this PR
Log in
Throttle your network
Load a draft project
You will see the content.

### CODE REVIEW

**Review by AlfredRosenthal — APPROVED** [review]
🐯

File: app/src/features/projects/create/CreateProjectPage.test.tsx [hunk 1]
```diff
@@ -472,15 +472,15 @@ describe('CreateProjectPage', () => {
       });
     });
 
-    it('calls the createDraft/updateDraft functions with WIP form data', async () => {
+    it.skip('calls the createDraft/updateDraft functions with WIP form data', async () => {
```
Comment (curtisupshall) [inline app/src/features/projects/create/CreateProjectPage.test.tsx:475]: Nit: test skip

**Review by NickPhura — APPROVED** [review]
Ran it locally, functionality from the users perspective seems happy.

File: app/src/features/projects/create/CreateProjectPage.test.tsx (hunk 1 above)
Comment (anissa-agahchen) [reply app/src/features/projects/create/CreateProjectPage.test.tsx:475]: addressing this in the next PR

**Pull request merged**